src/expressions/xquery-update/UpdatingExpression.ts

```diff
@@ -16,39 +47,11 @@ abstract class UpdatingExpression extends Expression {
 		this.isUpdating = true;
 	}
 
-	public ensureUpdateListWrapper(
```

Comment:
Any reason to move this function from the class?
I think we could have made it protected instead of public.